From 3ed99af4d5d0c4f2530290b96cdf33efb2ff32f0 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:42:36 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(566f231)..lineage-16.0-android-9.0.0_r31(9e6af80) (DO NOT
 SUBMIT)

Bill Yi (1):
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from 6e6aadd77ea07f3b2f1340e5c754b13b0cfcad09 to
   pi-qpr2-release

Change-Id: Ib837ba5223f4f39b23b9475b2bfb46bd143878a0

Change-Id: I06f7060ce882142249eaa7e9408ba390ed5333a0
---
 res/values-b+sr+Latn/strings.xml | 2 +-
 res/values-nl/strings.xml        | 2 +-
 res/values-pt/strings.xml        | 6 +++---
 res/values-sr/strings.xml        | 2 +-
 4 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/res/values-b+sr+Latn/strings.xml b/res/values-b+sr+Latn/strings.xml
index 8af507f..10dc7d8 100644
--- a/res/values-b+sr+Latn/strings.xml
+++ b/res/values-b+sr+Latn/strings.xml
@@ -29,7 +29,7 @@
     <string name="alphabet" msgid="1583185545938805861">"Abeceda"</string>
     <string name="digits" msgid="1746796679021682388">"Cifre (0-9, *, #, +)"</string>
     <string name="default_call_setup_msg" msgid="7960194995759849940">"Pozivanje je u toku..."</string>
-    <string name="default_setup_call_msg" msgid="6126904475461162162">"Poziv se podešava"</string>
+    <string name="default_setup_call_msg" msgid="6126904475461162162">"Poziv se pokreće"</string>
     <string name="stk_app_state" msgid="18582277302584082">"Stanje aplikacije"</string>
     <string name="enable_app" msgid="3701224550936728203">"Omogućeno"</string>
     <string name="disable_app" msgid="6725081975611415214">"Onemogućeno"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 9d34917..5ed9a87 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -28,7 +28,7 @@
     <string name="button_no" msgid="1824959157582605627">"Nee"</string>
     <string name="alphabet" msgid="1583185545938805861">"Alfabetten"</string>
     <string name="digits" msgid="1746796679021682388">"Symbolen (0-9, *, #, +)"</string>
-    <string name="default_call_setup_msg" msgid="7960194995759849940">"Actieve oproep..."</string>
+    <string name="default_call_setup_msg" msgid="7960194995759849940">"Actief gesprek..."</string>
     <string name="default_setup_call_msg" msgid="6126904475461162162">"Aanroep wordt ingesteld"</string>
     <string name="stk_app_state" msgid="18582277302584082">"Toepassingsstatus"</string>
     <string name="enable_app" msgid="3701224550936728203">"Ingeschakeld"</string>
diff --git a/res/values-pt/strings.xml b/res/values-pt/strings.xml
index 1cd5f21..fbd1cc6 100644
--- a/res/values-pt/strings.xml
+++ b/res/values-pt/strings.xml
@@ -16,7 +16,7 @@
 
 <resources xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
-    <string name="app_name" msgid="8396461461738434341">"Toolkit do SIM"</string>
+    <string name="app_name" msgid="8396461461738434341">"Toolkit do chip"</string>
     <string name="menu_end_session" msgid="7368541737750982217">"Finalizar a sessão"</string>
     <string name="help" msgid="9209980881219397319">"Ajuda"</string>
     <string name="menu_back" msgid="301227740977453364">"Voltar"</string>
@@ -33,7 +33,7 @@
     <string name="stk_app_state" msgid="18582277302584082">"Estado do app"</string>
     <string name="enable_app" msgid="3701224550936728203">"Ativado"</string>
     <string name="disable_app" msgid="6725081975611415214">"Desativado"</string>
-    <string name="stk_dialog_title" msgid="6954825385456886726">"Toolkit do SIM"</string>
+    <string name="stk_dialog_title" msgid="6954825385456886726">"Toolkit do chip"</string>
     <string name="default_tone_dialog_msg" msgid="4595366992944391641">"Reproduzindo tom"</string>
     <string name="default_open_channel_msg" msgid="2216070254100295924">"Abrir canal?"</string>
     <string name="default_send_data_msg" msgid="6011219698689931272">"Enviando dados"</string>
@@ -41,6 +41,6 @@
     <string name="default_close_channel_msg" msgid="765364262263839824">"Canal fechado"</string>
     <string name="stk_dialog_accept" msgid="8498901537508923727">"SIM"</string>
     <string name="stk_dialog_reject" msgid="921848059485746796">"NÃO"</string>
-    <string name="no_sim_card_inserted" msgid="5316620398365235988">"Insira o cartão SIM para iniciar a Barra de ferramentas SIM."</string>
+    <string name="no_sim_card_inserted" msgid="5316620398365235988">"Insira o chip para iniciar a Barra de ferramentas SIM."</string>
     <string name="stk_channel_name" msgid="8828587788561382056">"Mensagens de serviço móvel"</string>
 </resources>
diff --git a/res/values-sr/strings.xml b/res/values-sr/strings.xml
index 0438132..644e789 100644
--- a/res/values-sr/strings.xml
+++ b/res/values-sr/strings.xml
@@ -29,7 +29,7 @@
     <string name="alphabet" msgid="1583185545938805861">"Абецеда"</string>
     <string name="digits" msgid="1746796679021682388">"Цифре (0-9, *, #, +)"</string>
     <string name="default_call_setup_msg" msgid="7960194995759849940">"Позивање је у току..."</string>
-    <string name="default_setup_call_msg" msgid="6126904475461162162">"Позив се подешава"</string>
+    <string name="default_setup_call_msg" msgid="6126904475461162162">"Позив се покреће"</string>
     <string name="stk_app_state" msgid="18582277302584082">"Стање апликације"</string>
     <string name="enable_app" msgid="3701224550936728203">"Омогућено"</string>
     <string name="disable_app" msgid="6725081975611415214">"Онемогућено"</string>
-- 
2.17.1

